discovery-provider/alembic/versions/7b843f2d3a0d_load_audio_transactions_csv.py

```diff
@@ -0,0 +1,59 @@
+"""load audio transactions csv
+
+Revision ID: 7b843f2d3a0d
+Revises: 21e5635c83a9
+Create Date: 2022-12-05 20:30:57.288526
+
+"""
+import os
+import urllib.request
+import zipfile
+from pathlib import Path
+
+import sqlalchemy as sa
+from alembic import op
+
+# revision identifiers, used by Alembic.
+revision = "7b843f2d3a0d"
+down_revision = "21e5635c83a9"
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    env = os.getenv("audius_discprov_env")
+    if env == "stage":
+        pass
+
+    connection = op.get_bind()
+    query = """
+        delete from audio_transactions_history where slot < 164000000;
+    """
+    connection.execute(query)
+
+    aws_url = "https://s3.us-west-1.amazonaws.com/download.audius.co/audio_transactions_history.csv.zip"
+    path_zip = Path(__file__).parent.joinpath(
+        "../tmp/audio_transactions_history.csv.zip"
+    )
+    urllib.request.urlretrieve(aws_url, path_zip)
```

Comment:
nit: can we add a log around urlretrieve so when it looks like migrations are hanging, we know what's going on?